XenAdmin/Wizards/BugToolWizardFiles/BugToolPageSelectCapabilities.Designer.cs

```diff
@@ -43,14 +43,11 @@ private void InitializeComponent()
             this.DescriptionValue = new System.Windows.Forms.Label();
             this.SizeLabel = new System.Windows.Forms.Label();
             this.SizeValue = new System.Windows.Forms.Label();
-            this.TimeLabel = new System.Windows.Forms.Label();
-            this.TimeValue = new System.Windows.Forms.Label();
             this.TotalSizeLabel = new System.Windows.Forms.Label();
             this.TotalSizeValue = new System.Windows.Forms.Label();
-            this.TotalTimeLabel = new System.Windows.Forms.Label();
-            this.TotalTimeValue = new System.Windows.Forms.Label();
             this.tableLayoutPanel3 = new System.Windows.Forms.TableLayoutPanel();
             this.label1 = new System.Windows.Forms.Label();

```

Comment:
We need to also update the text of label1 because it still says you can see the estimated time.